packages/ag-charts-community/src/chart/chartAxesTypes.ts

```diff
@@ -12,7 +12,7 @@ const AXES_THEME_TEMPLATES: Record<string, {}> = {};
 
 export const CHART_AXES_TYPES = {
     has(axisType: string) {
-        return Object.prototype.hasOwnProperty.call(TYPES, axisType);
+        return Object.hasOwn(TYPES, axisType);
```

Comment:
What's our browser support?

Reply:
we compile to `es2015` but locally work on `es2022`, so it doesn't effect the output - only our code